extension/lib/commands.coffee

```diff
@@ -311,7 +311,7 @@ absoluteTabIndex = (relativeIndex, gBrowser, {pinnedSeparate}) ->
   currentIndex = tabs.indexOf(selectedTab)
   absoluteIndex = currentIndex + relativeIndex
   numTabsTotal = tabs.length
-  numPinnedTabs = gBrowser._numPinnedTabs
+  numPinnedTabs = gBrowser.pinnedTabCount or gBrowser._numPinnedTabs # fx132
```

Comment:
```suggestion
  numPinnedTabs = gBrowser.pinnedTabCount ? gBrowser._numPinnedTabs # fx132
```

You probably want `?` rather than `or` so that `numPinnedTabs` does not end up being `undefined` when `gBrowser.pinnedTabCount` is `0`.

[Demo](http://coffeescript.org/#try:gBrowser%20%3D%20%7B%20pinnedTabCount%3A%200%20%7D%0A%0AnumPinnedTabs%20%3D%20gBrowser.pinnedTabCount%20or%20gBrowser._numPinnedTabs%0Aconsole.log(%22numPinnedTabs%2C%20expected%200%20but%20is%20undefined%3A%22%2C%20numPinnedTabs)%0A%0AnumPinnedTabs%20%3D%20gBrowser.pinnedTabCount%20%3F%20gBrowser._numPinnedTabs%0Aconsole.log(%22numPinnedTabs%2C%20is%200%20as%20expected%3A%22%2C%20numPinnedTabs)%0A%0Adelete%20gBrowser.pinnedTabCount%0AgBrowser._numPinnedTabs%20%3D%201%0AnumPinnedTabs%20%3D%20gBrowser.pinnedTabCount%20%3F%20gBrowser._numPinnedTabs%0Aconsole.log(%22numPinnedTabs%20via%20old%20property%2C%20is%201%20as%20expected%3A%22%2C%20numPinnedTabs)%0A)